data/mods/Magiclysm/Spells/attunements/Magnetism_Mage.json

```diff
@@ -328,9 +289,10 @@
     "id": "robot_disabler_explosion",
     "type": "SPELL",
     "name": "Surge",
-    "description": "Send a wave of high-frequency alternating pulse.  It can't hurt really much, but it works great against circuits, shutting them down for a small time.  Probably it will work against drones or robots.  No, it's not EMP.",
+    "description": "Send a wave of high-frequency alternating pulse.  It can't hurt living creatures, but extremely good in destroying and stunning robots.  No, it's not EMP.",
```

Comment:
```suggestion
    "description": "Send a wave of high-frequency alternating pulse.  It can't hurt living creatures, but is extremely good destroying and stunning robots.  No, it's not EMP.",
```